tests/network/networkpolicy.go

```diff
@@ -275,8 +276,17 @@ var _ = Describe(SIG("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:co
 }))
 
 func assertPingSucceed(fromVmi, toVmi *v1.VirtualMachineInstance) {
+	var isMasquerade bool
+	for _, iface := range toVmi.Spec.Domain.Devices.Interfaces {
+		if iface.Masquerade != nil {
+			isMasquerade = true
+		}
+	}
 	ConsistentlyWithOffset(1, func() error {
 		for _, toIp := range toVmi.Status.Interfaces[0].IPs {
+			if toIp == cloudinit.DefaultIPv6Address && isMasquerade {
+				continue
+			}
```

Comment:
Since we have the masquerade IPv6 address only for VMs using masquerade binding (and all the VMs in this file use it), I think we can simplify this condition to just:
```suggestion
			if toIp == cloudinit.DefaultIPv6Address {
				continue
			}
```

Reply:
Ack, updated [here](https://github.com/kubevirt/kubevirt/compare/158a84f611234227b52d9c9fdfea3ec2aee7b06c..5bcc0285422e0cb7f2918bc9907d676a5f0b342f)